p1

```diff
@@ -0,0 +1,69 @@
+command_parse.c:1366:18: mutt_pretty_mailbox(buf->data, buf->dsize);
```

Comment:
I guess this did sneak in by accident?

Reply:
er, sort of.
The whole PR is still a work-in-progress and `p1` are notes of what I was doing last.
I have so many tiny temporary files like this, that I've started committing them so I don't lose them.